garrysmod/gamemodes/sandbox/gamemode/cl_hints.lua

```diff
@@ -1,62 +1,80 @@
 
+local cl_showhints = CreateConVar( "cl_showhints", "1",
+	bit.bor( FCVAR_ARCHIVE, FCVAR_USERINFO ), "Show hints called thru SANDBOX:AddHint", 0, 1 )
 
-CreateClientConVar( "cl_showhints", "1", true, false )
+-- A list of hints we've already done so we don't repeat ourselves
+GM.ProcessedHints = {}

```

Comment:
What is the point of exposing an internal table to public?

Reply:
This was to introduce the auto-refresh table idiom to the file but I forgot to write the full statement.